### ISSUE TITLE
[Desktop] Crash Report

### ISSUE THREAD

**Issue by FixmyBrave** [brave/brave-browser#11626]
IMPORTANT: Your crash has already been automatically reported to our crash system. Please file this bug only if you can provide more information about it.

Brave Version: 1.13.82  Chromium: 85.0.4183.83
Operating System: Mac OS X 10.15.6

URL (if applicable) where crash occurred:

Can you reproduce this crash?

What steps will reproduce this crash? (If it's not reproducible, what were you doing just before the crash?)
1. It happened often when in Google drive I would control click (right click) a docx file and go to open in Google Docs. So now I have to use Safari for my Google work (much of my work) and Brave crashes much less frequently. 
2. The above is the only repeatable action that caused the crash. It sometimes crashes when I just click a tab, when I try to load brave://crashes, when I click a link...I can do almost anything and it could crash.
3. I noticed that if it crashes and I reload and then close some apps that it seems more stable but I need those apps open to work. Apps include: terminal, TextEdit, PanelEditor (customer keyboard for a macro), Finder (2 windows). I haven't identified which app(s) being closed help anything.

****DO NOT CHANGE BELOW THIS LINE****
Crash ID: crash/e7220000-88e8-0505-0000-000000000000

**Comment by bsclifton** [comment]
Curious if this is a dupe of https://github.com/brave/brave-browser/issues/11812

**Comment by rebron** [comment]
Hi @FixmyBrave We're wondering if you were you using an onscreen keyboard at any point when you crashed?

**Comment by FixmyBrave** [comment]
Yes @rebron  but it was a custom keyboard I created. 2 or 3 buttons (I added one at some point) that each acted like a macro, performing several steps. Each time the keyboard was open. The crashed didn't occur while using the keyboard though. It could crash doing basically anything. Open a new tab, open settings, try to open brave://crashes, clicking a link on a site...literally any thing could either crash it or work fine. Worst game of russian roullette ever.
